src/src/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -127,7 +127,7 @@ public void onReceive(Context context, Intent intent) {
             Logger.v(TAG, "ActivityBroadcastReceiver onReceive");
 
             if (intent.getAction().equalsIgnoreCase(AuthenticationConstants.Browser.ACTION_CANCEL)) {
-                try {
+            //  try {
```

Comment:
I assume you are going to remove these lines.  Would you mind just doing that.  It clutters code reviews to see things that are in progress.